libs/payments/paypal/src/lib/paypal.manager.ts

```diff
@@ -3,10 +3,24 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { Injectable } from '@nestjs/common';
-import { PayPalClient } from './paypal.client';
+
 import { AccountDatabase } from '@fxa/shared/db/mysql/account';
 
+import { PayPalClient } from './paypal.client';
+import { SetExpressCheckoutOptions } from './types';
+
 @Injectable()
 export class PayPalManager {
   constructor(private db: AccountDatabase, private client: PayPalClient) {}
+
+  /**
+   * Get a token authorizing transaction to move to the next stage.
+   *
+   * If the call to PayPal fails, a PayPalClientError will be thrown.
+   *
+   */
+  async getCheckoutToken(options: SetExpressCheckoutOptions) {
```

Comment:
This layer should not expose the inner `SetExpressCheckoutOptions` type that comes from the client. Rather, it should just take currencyCode as a string.